noark-extraction-validator/pom.xml

```diff
@@ -54,13 +53,13 @@
 		</dependency>
 
 		<dependency>
-			<groupId>commons-configuration</groupId>
-			<artifactId>commons-configuration</artifactId>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-configuration2</artifactId>
```

Comment:
This new version of commons-configuration seam to require this package as well:
```
		<dependency>
			<groupId>commons-beanutils</groupId>
			<artifactId>commons-beanutils</artifactId>
			<version>1.9.4</version>
		</dependency>
```
otherwise we will hit the following issue when running the standalone jar:
```
Exception in thread "main" java.lang.NoClassDefFoundError: org/apache/commons/beanutils/BeanIntrospector
        at java.base/java.lang.Class.forName0(Native Method)
        at java.base/java.lang.Class.forName(Class.java:375)
        at jdk.proxy2/jdk.proxy2.$Proxy9.<clinit>(Unknown Source)
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
        at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
        at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
        at java.base/java.lang.reflect.Proxy.newProxyInstance(Proxy.java:1053)
        at java.base/java.lang.reflect.Proxy.newProxyInstance(Proxy.java:1039)
        at org.apache.commons.configuration2.builder.fluent.Parameters.createParametersProxy(Parameters.java:243)
        at org.apache.commons.configuration2.builder.fluent.Parameters.properties(Parameters.java:198)
        at com.documaster.validator.config.properties.InternalProperties.<init>(InternalProperties.java:46)
        at com.documaster.validator.config.properties.Noark5Properties.<init>(Noark5Properties.java:41)
        at com.documaster.validator.config.commands.Noark5Command.getProperties(Noark5Command.java:164)
        at com.documaster.validator.validation.Noark5Validator.validateStructure(Noark5Validator.java:211)
        at com.documaster.validator.validation.Noark5Validator.run(Noark5Validator.java:90)
        at com.documaster.validator.main.Run.main(Run.java:51)
Caused by: java.lang.ClassNotFoundException: org.apache.commons.beanutils.BeanIntrospector
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
        ... 18 more
```

Reply:
Addressed w/ https://github.com/documaster/noark-extraction-validator/pull/66/commits/b94c3fa94913345f901788b827972196eca60618